code/modules/mob/mob_grab.dm

```diff
@@ -37,7 +37,7 @@
 
 
 //Used by throw code to hand over the mob, instead of throwing the grab. The grab is then deleted by the throw code.
-/obj/item/weapon/grab/proc/throw()
+/obj/item/weapon/grab/proc/throw_target()

```

Comment:
As far as I can tell? BYOND decided throw() was now a reserved proc name and refused to allow it to compile without altering it. I've also changed all calls to the procedure, so everything should fine.